digits/model/tasks/caffe_train.py

```diff
@@ -798,15 +806,42 @@ def task_arguments(self, resources, env):
                 identifiers.append(identifier)
             if len(identifiers) == 1:
                 args.append('--gpu=%s' % identifiers[0])
+                if win_python_layer:
+                    win_python_layer_gpu_id = identifiers[0]
```

Comment:
I thought we agreed to throw an error here instead of silently ignoring the other GPUs?